src/clj/backend/main.clj

```diff
@@ -1,21 +1,25 @@
 (ns backend.main
   (:require [aero.core :as aero]
             [backend.routes]
-            [clojure.java.io :as io]
             [clojure.tools.logging :as log]
             [cognitect.transit]
             [hikari-cp.core :as hikari-cp]
             [integrant.core :as ig]
             [next.jdbc.date-time]
             [reitit.ring.middleware.exception]
-            [ring.adapter.jetty :as jetty]))
+            [ring.adapter.jetty9 :as jetty]
+            #_[ring.adapter.jetty :as jetty])
+  (:gen-class))
+
+(set! *warn-on-reflection* true)
 
 (defmethod aero.core/reader 'ig/ref
   [_opts _tag value]
   (ig/ref value))
 
 (defn system-config []
-  (aero/read-config (io/resource "config.edn")))
+  (aero/read-config (or (System/getenv "CONFIG_EDN")
+                        "config.edn")))
```

Comment:
I think adding the env var here does make sense, but AFAIK removing the `io/resource` call does break the case of reading the file from classpath.